src/server/server_family.cc

```diff
@@ -1392,11 +1396,25 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
 
     auto unknown_cmd = service_.UknownCmdMap();
 
+    vector<pair<string_view, uint64_t>> display;
+    display.reserve(std::max(unknown_cmd.size(), m.conn_stats.cmd_count_map.size()));
+
     for (const auto& k_v : unknown_cmd) {
+      display.push_back(k_v);
+    }
+    sort(display.begin(), display.end());
+
+    for (const auto& k_v : display) {
```

Comment:
Extract to function AppendSorted(... ?

Reply:
Fixed

Reply:
I meant the entire thing :)

Also I'd move to a non lambda that does
reserve 
push each
append each

It is an extra reserve call but much more readable IMHO

Reply:
It's a bit problematic because the source maps have a bit different types.  Pulling lambda out will create a weird template function that is used in a single place - here.  I thought it's not worth the complexity.

Reply:
in matter of fact, `append` is also a lambda function so it will be pretty complicated to introduce something well defined outside of the scope of Info function.